### ISSUE TITLE
Autowired spring beans are recognized as variables that could be local

### ISSUE THREAD

**Issue by BalandinArtem** [spotbugs/sonar-findbugs#273]
## Issue Description

Since updating to a new sonarqube version I have faced with issue:
sonar noted my spring autowired beans as a vars that could be a local method vars.
Code example and versions are added below

Rule caused a problem: 
'''
This class defines fields that are used in a local only fashion, specifically private fields or protected fields in final classes that are accessed first in each method with a store vs. a load. This field could be replaced by one or more local variables.
fb-contrib:FCBL_FIELD_COULD_BE_LOCAL
Correctness - Class defines fields that are used only as locals
'''

## Environment

| Component          | Version |
| ------------------ | ------- |
| SonarQube          | 7.7.0.23042   |
| Sonar-FindBugs     | 3.11.0   |
| Maven              | 3.6   |
| Java               | jdk11  |

## Code (If needed)

```java
@Component
public class TextClass {

    @Autowired private TestBean testBean;

    .....
}
```

**Comment by gtoison** [comment]
Sorry for the belated answer, this false positive should be reported to the [fb-contrib project](https://github.com/mebigfatguy/fb-contrib)
